MassMessage/Scripts/sendMessage.ts

```diff
@@ -10,17 +10,22 @@ const authToken = process.env.TWILIO_AUTH_TOKEN
 const client = require('twilio')(accountSid, authToken)
 
 //will need to update functions when phone number is not directly stored
+//another update for when multimedia messages get added, current both temporary string placeholders
 
-export const sendCompletedMessage = async function (id: string, date: string, modules: Array<number>) {
-    await MongoConnect()
-    const message = `Hi, you completed module(s) ${modules.join()} on ${date}. Pls respond.`
+export const sendCompletedMessage = async function (id: string, date: string, modules: Array<number>, lowData: boolean) {
+    await MongoConnect();
+
+    const message = lowData ? `Hi but on low data, you completed module(s) ${modules.join()} on ${date}. Pls respond.` 
+        : `Hi, you completed module(s) ${modules.join()} on ${date}. Pls respond.`;
 
     await client.messages.create({ body: message, from: '+13159300241', to: id })
 }
 
-export const sendInactiveMessage = async function (id: string, date: string) {
-    await MongoConnect()
-    const message = `Hi, you haven't had any activity since ${date}. Pls respond.`
+export const sendInactiveMessage = async function (id: string, date: string, lowData: boolean) {
+    await MongoConnect();
+
+    const message = lowData ? `Hi but on low data, you haven't had any activity since ${date}. Pls respond.` 
+    : `Hi, you haven't had any activity since ${date}. Pls respond.`;
```

Comment:
For these messages, we no longer need to write messages for these. Use the ones present in the fixed messages. Also be sure to set the userState properly; currently that is outside of this function in `index.ts`, but it's very likely that we need to return the proper message from the fixed message id so that we can access it in `index.ts` and set and save `userState`'s `currMessage` to that fixed message.